defmt/src/lib.rs

```diff
@@ -367,6 +367,42 @@ pub use defmt_macros::timestamp;
 /// ```
 pub use defmt_macros::bitflags;
 
+/// Users of the defmt crate can enable the bitflagsv2 feature by adding defmt = { features = ["bitflagsv2"] } 
+/// to their Cargo.toml. Bitflags version 2 introduces significant improvements over version 1, including a safer 
+/// API with the replacement of the unsafe from_bits_unchecked method by the safe from_bits_retain method 
+/// and enhanced serialization support via an optional serde feature.
+///
+/// This macro is a wrapper around the [`bitflags!`][bitflagsv2] macro of the bitflags! version 2 crate, and provides an (almost) identical
```

Comment:
Nit pick: the macro name can have an `!` in it, the crate should not. So please change `bitflags! version 2 crate` to `bitflags version 2 crate`.
The `bitflags! macro` can keep the `!`